src/components/menu/AudioDescription.js

```diff
@@ -0,0 +1,78 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  player: PropTypes.object,
+  actions: PropTypes.object,
+  className: PropTypes.string,
+  audioDescriptions: PropTypes.any
```

Comment:
Non-blocking, and I don't feel strongly about this, but have you considered  `PropTypes.arrayOf(PropTypes.shape({/* ... */}))` here? `PropTypes.any` seems rather vaguer than necessary.